terraform/modules/hub/files/prometheus/alerts.yml

```diff
@@ -14,31 +14,75 @@ groups:
       severity: "constant"
 - name: service
   rules:
-  - alert: HubSamlProxyNonEidasErrorRate
+  - alert: HubSamlProxyErrorsReceivingRequest
     labels:
       severity: "p1"
     annotations:
       message: |
-        It looks like users are having trouble with SAML interactions
-        in their journeys at the hub.  This means that they will be
-        unable to complete their journeys.  We expect that the
-        saml-proxy endpoints should return a 2xx response under normal
-        conditions.  We are observing the rate of 2xx responses has
-        dropped below 95%.
+        It looks like users are having trouble starting sessions at
+        the hub.  We expect that the saml-proxy handleRequestPost
+        endpoint should return a 2xx response under normal conditions.
+        We are observing the rate of 2xx responses has dropped below
```

Comment:
My point is slightly pedantic, and I assume irrelevant for what saml-proxy does, but anyway: we should count 3xx as successful too? Really only 4xx and 5xx should count as errors?

Reply:
SAML Proxy does not return `3xx`s